static/etc/system-image/writable-paths

```diff
@@ -16,10 +16,10 @@
 # snap data
 /var/snap                               auto                    persistent  transition  none
 # generic
-/media                                  none                    temporary   none        defaults
-/mnt                                    none                    temporary   none        defaults
+/media                                  none                    temporary   none        mode=1777,strictatime,nosuid,nodev
```

Comment:
Do we need `mode=1777` for /mnt and /media ? Those are not really dirs that have the same semantic as /tmp, they are more dirs with a similar semantic as /run AIUI. Or is there soemthing I'm missing here?

Reply:
Without this change, when I run `stat` on `/mnt` and `/media` it actually shows mode `1777` already, so I think that is the default mode. This just shows that more explicitly. So we can remove it I guess, I do not have a strong opinion either way.